app/controllers/alchemy/admin/change_site_controller.rb

```diff
@@ -0,0 +1,17 @@
+module Alchemy
+  module Admin
+    class ChangeSiteController < BaseController
+      def index
+        authorize! :select, :alchemy_admin_site
+
+        session[:site_id] = requested_site_id
+        do_redirect_to params[:redirect_to]
+      end
+
+      private
```

Comment:
Style/EmptyLinesAroundAccessModifier: Keep a blank line before and after `private`.